packages/x-date-pickers/src/PickersLayout/PickersLayout.tsx

```diff
@@ -84,8 +84,17 @@ const PickersLayout = function PickersLayout<TValue, TView extends DateOrTimeVie
     >
       {toolbar}
       <PickersLayoutContentWrapper className={classes.contentWrapper}>
-        {tabs}
-        {content}
+        {wrapperVariant === 'desktop' ? (
```

Comment:
I don't use the array format a lot with JSX, but with variables like this it's lighter imho:

```suggestion
        {wrapperVariant === 'desktop' ? [content, tabs] : [tabs, content]}
```

Reply:
Sadly, doing this produces a missing `key` warning.
I'd love this much cleaner approach, but it's not easy to apply it in this case. 🤔 
<img width="715" alt="Screenshot 2023-01-09 at 10 27 29" src="https://user-images.githubusercontent.com/4941090/211266672-2189efa0-057e-48f8-8472-cc5b8f6fe6d1.png">